### ISSUE TITLE
Dashboard Click Custom Destination doesn't work if any filter is null

### ISSUE THREAD

**Issue by gustavotrott** [metabase/metabase#16201]
**Describe the bug**
I created an element in the Dashboard and change the "Click behavior", linked to a question and set some filters to pass dashboard filter to the question when click in the element.
But the link doesn't work if one of the filters is null. It enables the link/click only after I fill all dashboard filters. But sometimes I want to keep them null and thus don't pass this filter to the question.

**To Reproduce**

1. Create a Dashboard with filters
2. Include a number element in the Dashboard
3. Change the "Click behavior" and link it to a Question
4. Select some columns of the question to link with Dashboard filters
5. Keep the dashboard filter null
6. Click in the element
7. Nothing will happen

**Comment by flamber** [comment]
Hi @gustavotrott 
Please post "Diagnostic Info" from Admin > Troubleshooting.
I cannot reproduce. Can you provide steps-to-reproduce on the Sample Dataset?

**Comment by gustavotrott** [comment]
Hey @flamber,

It happens when I have two date filters in Dashboard.. 
For example

- One linked to "Orders -> Created at"
- Seconded linked to "Product -> Created at"

When I click in the number, it should open a question including this filters.
But the second is null, the click in the number doesn't work.
![image](https://user-images.githubusercontent.com/5660191/120332038-bb7f6900-c2c4-11eb-9dcf-bbe3c54826a8.png)

If I fill both, it works:
![image](https://user-images.githubusercontent.com/5660191/120332307-01d4c800-c2c5-11eb-89a5-2796845135df.png)
![image](https://user-images.githubusercontent.com/5660191/120332324-05684f00-c2c5-11eb-9fbc-c4413aa5abf9.png)


I am trying to figure out the exactly scenario to reproduce this.. because this issue is quite inconstant.
If both filters are null, then the click works..

Next days I will provide better steps to reproduce.

**Comment by flamber** [comment]
@gustavotrott Please post "Diagnostic Info" from Admin > Troubleshooting. Latest release is 0.39.3

**Comment by gustavotrott** [comment]
- Desenv

```
{
  "browser-info": {
    "language": "pt-BR",
    "platform": "Win32",
    "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:88.0) Gecko/20100101 Firefox/88.0",
    "vendor": ""
  },
  "system-info": {
    "file.encoding": "Cp1252",
    "java.runtime.name": "Java(TM) SE Runtime Environment",
    "java.runtime.version": "1.8.0_291-b10",
    "java.vendor": "Oracle Corporation",
    "java.vendor.url": "http://java.oracle.com/",
    "java.version": "1.8.0_291",
    "java.vm.name": "Java HotSpot(TM) 64-Bit Server VM",
    "java.vm.version": "25.291-b10",
    "os.name": "Windows 10",
    "os.version": "10.0",
    "user.language": "pt",
    "user.timezone": "America/Sao_Paulo"
  },
  "metabase-info": {
    "databases": [
      "postgres",
      "h2"
    ],
    "hosting-env": "unknown",
    "application-database": "postgres",
    "application-database-details": {
      "database": {
        "name": "PostgreSQL",
        "version": "12.3"
      },
      "jdbc-driver": {
        "name": "PostgreSQL JDBC Driver",
        "version": "42.2.18"
      }
    },
    "run-mode": "prod",
    "version": {
      "tag": "v0.39.3",
      "date": "2021-05-27",
      "branch": "release-x.39.x",
      "hash": "598a112"
    },
    "settings": {
      "report-timezone": null
    }
  }
}
```

- Prod
```
{
  "browser-info": {
    "language": "pt-BR",
    "platform": "Win32",
    "userAgent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:88.0) Gecko/20100101 Firefox/88.0",
    "vendor": ""
  },
  "system-info": {
    "file.encoding": "UTF-8",
    "java.runtime.name": "OpenJDK Runtime Environment",
    "java.runtime.version": "1.8.0_181-b13",
    "java.vendor": "Oracle Corporation",
    "java.vendor.url": "http://java.oracle.com/",
    "java.version": "1.8.0_181",
    "java.vm.name": "OpenJDK 64-Bit Server VM",
    "java.vm.version": "25.181-b13",
    "os.name": "Linux",
    "os.version": "3.10.0-514.10.2.el7.x86_64",
    "user.language": "en",
    "user.timezone": "America/Sao_Paulo"
  },
  "metabase-info": {
    "databases": [
      "postgres"
    ],
    "hosting-env": "unknown",
    "application-database": "postgres",
    "application-database-details": {
      "database": {
        "name": "PostgreSQL",
        "version": "9.3.16"
      },
      "jdbc-driver": {
        "name": "PostgreSQL JDBC Driver",
        "version": "42.2.18"
      }
    },
    "run-mode": "prod",
    "version": {
      "tag": "v0.39.3",
      "date": "2021-05-27",
      "branch": "release-x.39.x",
      "hash": "598a112"
    },
    "settings": {
      "report-timezone": "America/Sao_Paulo"
    }
  }
```

**Comment by flamber** [comment]
@gustavotrott I have not been able to reproduce. Please check the browser console for errors, when the click doesn't work.

**Comment by gustavotrott** [comment]
Hi @flamber,

When I open the dashboard with one filter filled and other unfilled, I got several times the same error..

- Firefox
![image](https://user-images.githubusercontent.com/5660191/120727106-10cb9e00-c4b0-11eb-9518-a6dc9f533953.png)

- Chrome
![image](https://user-images.githubusercontent.com/5660191/120727111-1628e880-c4b0-11eb-99a7-6ee252d24c8f.png)

Not sure if this prints can help..
Let me know if I can send any detail more useful!

Or maybe I can configure some environment you can access and check.

**Comment by flamber** [comment]
@gustavotrott I think a video showing the entire process you are doing on a new dashboard, or access to an instance is the only way. I have tried several different things based on your steps-to-reproduce, but I cannot reproduce, so either you're missing a step or it's something to do with "this issue is quite inconstant", which I guess means that sometimes it works, sometimes it doesn't (in other words, completely random).

**Comment by gustavotrott** [comment]
Sure, @flamber!
I just started Metabase using the default database.

https://user-images.githubusercontent.com/5660191/121565689-14e94580-c9f3-11eb-80af-ad37b1a60921.mp4

**Comment by flamber** [comment]
@gustavotrott Awesome video :heart:
So this only triggers, when both filters are Time, and only when the first filter is defined and targeting a question. If only the second filter is defined, then it works. This is slightly puzzling, but I don't fully understand the logic behind the generated links.

Reproduce:
1. Simple question > Sample Dataset > Orders - save as "Q1"
2. Simple question > Sample Dataset > Orders - summarize Count - save as "Q2" and add to dashboard
3. On dashboard, add two Time:AllOptions filters - call then "Orders" and "Products" and connect to respective CreatedAt column on "Q2"
![image](https://user-images.githubusercontent.com/1447303/121676485-fcd9fa80-cab4-11eb-9a98-ba64e800f5fb.png)
4. Hover "Q2" > Click Behavior > Custom destination Question > select "Q1" and connect the columns to the filters
![image](https://user-images.githubusercontent.com/1447303/121676655-298e1200-cab5-11eb-9a1d-287c97c60f1b.png)
5. Save dashboard and input "Past 30 Years" into the first ("Orders") filter and click the "Q2" value - nothing happens and browser console doesn't produce new errors/warnings.
![image](https://user-images.githubusercontent.com/1447303/121676848-6c4fea00-cab5-11eb-9272-7c5dfa1e8c96.png)

Doing a browser refresh will show a lot of errors in the browser console indicating a problem of a null field, which likely causes problems later on, when it tries to generate the URL for the click behavior:
<details><summary>Browser console errors</summary>

```
app-main.bundle.js?8f423c47488b32b1f66b:5 Setting moment.js start of week for Locale en to monday
app-main.bundle.js?8f423c47488b32b1f66b:5 DEPRECATED: metabase/redux/metadata addFields
y @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 DEPRECATED: metabase/redux/metadata fetchField
y @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 DEPRECATED: metabase/redux/metadata fetchFieldValues
y @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 Unknown MBQL Field clause null
value @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 DEPRECATED: metabase/redux/metadata fetchField
y @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 DEPRECATED: metabase/redux/metadata fetchFieldValues
y @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 Error getting setting scalar.field TypeError: Cannot read property 'cols' of undefined
    at Object.getDefault (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at _ (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t.getComputedSettings (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at h (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at Mo (vendor.bundle.js?8f423c47488b32b1f66b:487)
    at Ia (vendor.bundle.js?8f423c47488b32b1f66b:487)
    at vs (vendor.bundle.js?8f423c47488b32b1f66b:487)
    at vu (vendor.bundle.js?8f423c47488b32b1f66b:487)
_ @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 Unknown MBQL Field clause null
value @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 Error getting setting scalar.field TypeError: Cannot read property 'cols' of undefined
    at Object.getDefault (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at _ (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t.getComputedSettings (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at h (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at Mo (vendor.bundle.js?8f423c47488b32b1f66b:487)
    at Ia (vendor.bundle.js?8f423c47488b32b1f66b:487)
    at vs (vendor.bundle.js?8f423c47488b32b1f66b:487)
    at vu (vendor.bundle.js?8f423c47488b32b1f66b:487)
_ @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 Unknown MBQL Field clause null
value @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 Unknown MBQL Field clause null
value @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 Unknown MBQL Field clause null
value @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 Unknown MBQL Field clause null
value @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 Unknown MBQL Field clause null
value @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 Unknown MBQL Field clause null
value @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 TypeError: object null is not iterable (cannot read property Symbol(Symbol.iterator))
    at Function.from (<anonymous>)
    at a (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at new t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at Array.map (<anonymous>)
    at t.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at i (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t.n.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
n.visualizationIsClickable @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 TypeError: object null is not iterable (cannot read property Symbol(Symbol.iterator))
    at Function.from (<anonymous>)
    at a (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at new t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at Array.map (<anonymous>)
    at t.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at i (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t.n.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
n.visualizationIsClickable @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 Unknown MBQL Field clause null
value @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 TypeError: object null is not iterable (cannot read property Symbol(Symbol.iterator))
    at Function.from (<anonymous>)
    at a (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at new t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at Array.map (<anonymous>)
    at t.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at i (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t.n.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
n.visualizationIsClickable @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 Unknown MBQL Field clause null
value @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 TypeError: object null is not iterable (cannot read property Symbol(Symbol.iterator))
    at Function.from (<anonymous>)
    at a (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at new t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at Array.map (<anonymous>)
    at t.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at i (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t.n.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
n.visualizationIsClickable @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 TypeError: object null is not iterable (cannot read property Symbol(Symbol.iterator))
    at Function.from (<anonymous>)
    at a (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at new t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at Array.map (<anonymous>)
    at t.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at i (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t.n.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
n.visualizationIsClickable @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 TypeError: object null is not iterable (cannot read property Symbol(Symbol.iterator))
    at Function.from (<anonymous>)
    at a (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at new t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at Array.map (<anonymous>)
    at t.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at i (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t.n.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
n.visualizationIsClickable @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 TypeError: object null is not iterable (cannot read property Symbol(Symbol.iterator))
    at Function.from (<anonymous>)
    at a (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at new t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at Array.map (<anonymous>)
    at t.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at i (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t.n.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
n.visualizationIsClickable @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 TypeError: object null is not iterable (cannot read property Symbol(Symbol.iterator))
    at Function.from (<anonymous>)
    at a (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at new t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at Array.map (<anonymous>)
    at t.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at i (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t.n.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
n.visualizationIsClickable @ app-main.bundle.js?8f423c47488b32b1f66b:5
vendor.bundle.js?8f423c47488b32b1f66b:487 Uncaught Error: Minified React error #188; visit https://reactjs.org/docs/error-decoder.html?invariant=188 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at vendor.bundle.js?8f423c47488b32b1f66b:487
    at tt (vendor.bundle.js?8f423c47488b32b1f66b:487)
    at Object.t.findDOMNode (vendor.bundle.js?8f423c47488b32b1f66b:487)
    at p.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at c (vendor.bundle.js?8f423c47488b32b1f66b:448)
vendor.bundle.js?8f423c47488b32b1f66b:487 Uncaught Error: Minified React error #188; visit https://reactjs.org/docs/error-decoder.html?invariant=188 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at vendor.bundle.js?8f423c47488b32b1f66b:487
    at tt (vendor.bundle.js?8f423c47488b32b1f66b:487)
    at Object.t.findDOMNode (vendor.bundle.js?8f423c47488b32b1f66b:487)
    at p.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at c (vendor.bundle.js?8f423c47488b32b1f66b:448)
app-main.bundle.js?8f423c47488b32b1f66b:5 TypeError: object null is not iterable (cannot read property Symbol(Symbol.iterator))
    at Function.from (<anonymous>)
    at a (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at new t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at Array.map (<anonymous>)
    at t.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at i (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t.n.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
n.visualizationIsClickable @ app-main.bundle.js?8f423c47488b32b1f66b:5
vendor.bundle.js?8f423c47488b32b1f66b:487 Uncaught Error: Minified React error #188; visit https://reactjs.org/docs/error-decoder.html?invariant=188 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at vendor.bundle.js?8f423c47488b32b1f66b:487
    at tt (vendor.bundle.js?8f423c47488b32b1f66b:487)
    at Object.t.findDOMNode (vendor.bundle.js?8f423c47488b32b1f66b:487)
    at p.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at p.<anonymous> (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at c (vendor.bundle.js?8f423c47488b32b1f66b:448)
app-main.bundle.js?8f423c47488b32b1f66b:5 TypeError: object null is not iterable (cannot read property Symbol(Symbol.iterator))
    at Function.from (<anonymous>)
    at a (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at new t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at Array.map (<anonymous>)
    at t.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at i (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t.n.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
n.visualizationIsClickable @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 TypeError: object null is not iterable (cannot read property Symbol(Symbol.iterator))
    at Function.from (<anonymous>)
    at a (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at new t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at Array.map (<anonymous>)
    at t.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at i (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t.n.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
n.visualizationIsClickable @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 TypeError: object null is not iterable (cannot read property Symbol(Symbol.iterator))
    at Function.from (<anonymous>)
    at a (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at new t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at Array.map (<anonymous>)
    at t.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at i (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t.n.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
n.visualizationIsClickable @ app-main.bundle.js?8f423c47488b32b1f66b:5
app-main.bundle.js?8f423c47488b32b1f66b:5 TypeError: object null is not iterable (cannot read property Symbol(Symbol.iterator))
    at Function.from (<anonymous>)
    at a (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at new t (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at Array.map (<anonymous>)
    at t.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at app-main.bundle.js?8f423c47488b32b1f66b:5
    at i (app-main.bundle.js?8f423c47488b32b1f66b:5)
    at t.n.value (app-main.bundle.js?8f423c47488b32b1f66b:5)
n.visualizationIsClickable @ app-main.bundle.js?8f423c47488b32b1f66b:5
```

</details>

**Comment by ranquild** [comment]
@flamber I couldn't repro the issue. Please take a look https://www.loom.com/share/24e309ba8bc44518b1c76c90c98e5aea

**Comment by flamber** [comment]
This was fixed during the filter revamp work already done in 0.42
